<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWizPreferenceWindow</class>
 <widget class="QDialog" name="CWizPreferenceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>411</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WizPreferenceDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxLang">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <widget class="QComboBox" name="comboLang">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxSkin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Skin</string>
     </property>
     <widget class="QComboBox" name="comboSkin">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonCheckUpgrade">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>check now</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReading">
    <attribute name="title">
     <string>Reading</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxNoteView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Default note view</string>
     </property>
     <widget class="QRadioButton" name="radioAuto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto (Keep view state)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioAlwaysEditing">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Always open note in Editing-View</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioAlwaysReading">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Always open note in Reading-View</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSyncing">
    <attribute name="title">
     <string>Syncing</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxSync">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Sync Method</string>
     </property>
     <widget class="QCheckBox" name="checkAutoSync">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto syncing</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkDownloadAllNotesData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>311</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Download all notes data</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelProxySettings">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Proxy Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
